src/lib/map/analysis/MapAnalysis.tsx

```diff
@@ -255,6 +246,12 @@ export function MapAnalysisImpl(props: Props & CompleteAppState) {
     studyMetadata.rootEntity.id,
   ]);
 
+  const totalEntitiesInSampleCount = (() => {
+    if (!totalCounts.value || !subsetVariableAndEntity.entityId) return 0;
+
+    return totalCounts.value[subsetVariableAndEntity.entityId];
+  })();
+
```

Comment:
Is `totalCounts.value[subsetVariableAndEntity.entityId]` have the data we're looking for?